subsys/bluetooth/controller/Kconfig

```diff
@@ -138,6 +138,19 @@ config BT_CTLR_SDC_MAX_CONN_EVENT_LEN_DEFAULT
 	  microseconds. The event length and the connection interval are the
 	  primary parameters for setting the throughput of a connection.
 
+config BT_CTLR_SDC_PERIODIC_ADV_EVENT_LEN_DEFAULT_OVERRIDE
```

Comment:
What is the reason for having the `_OVERRIDE` config? Can we not just set the default for `BT_CTLR_SDC_PERIODIC_ADV_EVENT_LEN_DEFAULT` unconditionally?

Reply:
We want to be able override the default out of tree for certain configurations

Reply:
That should be possible anyway as long as the out-of-tree config definition is found before this one. Though maybe simpler to keep it for now.

Reply:
> That should be possible anyway as long as the out-of-tree config definition is found before this one.

Unfortunately this is not always the case